.github/workflows/ci_main.yml

```diff
@@ -35,10 +35,8 @@ jobs:
       - name: rxjs dtslint
         run: yarn workspace rxjs dtslint
       - name: rxjs test:import
-        if: ${{ matrix.node == '18' || matrix.node == '20' }}
```

Comment:
I imagine these conditions originally distinguished these steps, but as it stands now we only run the jobs on v18 and v20 so the condition is redundant and I went ahead and cleaned them up